src/kompose-util.ts

```diff
@@ -16,8 +16,8 @@ const stableKomposeVersion = "v1.18.0";
 
 export async function getKomposePath() {
     var komposePath = "";
-    if (core.getInput('kompose-version', { required: false })) {
-        var version = core.getInput('kompose-version', { required: false });
+    var version = core.getInput('kompose-version', { required: false });
+    if (version) {
         if ( !!version && version != "latest" ){
```

Comment:
version should be refactored out to a constant variable somewhere. It's used a few times it looks like.